src/openms/source/ANALYSIS/ID/AScore.cpp

```diff
@@ -135,155 +153,157 @@ namespace OpenMS
         for (Size i = 0; i < 10; ++i)
```

Comment:
could we make this a parameter at least, this 10 peak rule seems quite crude to me...